### ISSUE TITLE
[Feature] Turn off map if using screen reader

### ISSUE THREAD

**Issue by hildeor** [AtB-AS/mittatb-app#640]
## Origin
_Links to received feedback, user research or other findings._

During the usability testing with Blindeforbundet a bug with Mapbox was discovered. On some iPhones (or different versions of iOS), the map crashes the app. So when trying to open Reisedetaljer, the whole app crashes, making Reisedetaljer completely unavailable. 

## Proposed solution
_A coarse description of a proposed solution, that may include wireframes and graphic design._

Probably dependent on Mapbox fixing this bug. Until then, suggest that we disable the map if a screen reader on iOS is detected.

**Comment by mikaelbr** [comment]
Thous should be fixed by #731, @cbrevik
